ucm2/Tegra/max98090/HiFi.conf

```diff
@@ -37,10 +37,38 @@ SectionVerb {
 	]
 }
 
+SectionDevice."Speakers" {
+	Comment = "Speakers"
+
+	ConflictingDevice [
+		"Headphones"
+	]
+	Value {
+		PlaybackPriority 100
+		PlaybackPCM "hw:${CardId}"
+		PlaybackVolume "Speaker Volume"
+	}
+	EnableSequence [
+		cset "name='Speakers Switch' on"
```

Comment:
Could you rewrite the configuration so:

1) all switches are off in the Verb.EnableSequence
2) manipulate only with the necessary controls in the Device sequences

Also, the default values should be moved from Verb.EnableSequence to the main file - BootSequence.

Reply:
Done, but i'm not sure i understand this step correctly:
> all switches are off in the Verb.EnableSequence